styles.css

```diff
@@ -0,0 +1,764 @@
+body {
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+    font-family: 'Bakbak One', Arial, sans-serif;
+    background: #0D0D15;
+}
+
+/* Bakbak One for headings */
+.bakbak-one-regular {
+    font-weight: 400;
+    font-style: normal;
+}
+
+/* Poppins for body text */
+.poppins-medium {
+    font-family: 'Poppins', Arial, sans-serif;
+    font-weight: 500;
+    font-style: normal;
+}
+
+/* Header */
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px 40px;
+    background-color: #0D0D15;
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+}
+
+.logo img {
+    height: 42px;
+    margin-right: 10px;
+}
+
+/* Navigation */
+
+.main-nav {
+    display: flex;
+    align-items: center;
+    vertical-align: top;
+    width: auto;
+}
+
+.main-nav__list {
+    display: flex;
+    list-style: none;
+}
+
+.main-nav__link {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+    text-decoration: none;
+    padding: 10px 15px;
+}
+
+.main-nav__list-item {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+    margin-right: 20px;
+}
+
+
+/* Hero Banner */
+.banner {
+    text-align: left;
+    padding: 50px 0;
+    background: radial-gradient(44.91% 42.19% at 22% -4%, rgba(2, 101, 220, 0.5), rgba(2, 101, 220, 0) 100%), rgb(13, 13, 21);
+    color: #F5F8FB;
+    font-size: 18px;
+    line-height: 1.5;
+}
+
+.banner__heading {
+    font-size: 48px;
+    margin-bottom: 20px;
+    line-height: 1.2;
+}
+
+.banner__highlight {
+    color: #0265DC;
+}
+
+.banner__text {
+    font-family: 'Poppins', sans-serif;
+    font-size: 18px;
+    margin-bottom: 30px;
+    line-height: 1.5;
+}
+
+.banner__stats {
+    display: block;
+    width: 100%;
+    max-width: 800px;
+}
+
+.banner__stat-label {
+    font-family: 'Poppins', sans-serif;
+    font-size: 16px;
+    line-height: 1.5;
+}
+
+.banner__stat-value {
+    font-size: 24px;
+}
+
+.banner__image {
+    height: calc(41vh - -93px);
+    margin-left: 86vh;
```

Comment:
Don't use vh unit for horizontal spacing 
For this image just set sizes in pixels from figma